### PR TITLE
LIMS-3257

### PR DESCRIPTION
**Pull request by omarkasem**

### CODE REVIEW

**Comment by 0xIslamTaha** [comment]
@omarkasem 
solve the conflict
image

**Comment by omarkasem** [comment]
@0xIslamTaha The conflicts are solved now

File: ui_testing/testcases/base_test.py [hunk 1]
```diff
@@ -19,6 +39,30 @@ def setUp(self):
...
+        self.test_plan_api = TestPlanAPI()
+        self.test_unit_api = TestUnitAPI()
+        self.test_plan_api = TestPlanAPI()
+        self.orders_api = OrdersAPI()
+        self.analysis_api = AnalysisAPI()
+        self.single_analysis_page = SingleAnalysisPage()
+        self.contacts_api = ContactsAPI()
+        self.users_api = UsersAPI()
+        self.roles_api = RolesAPI()
+        self.general_utilities_api = GeneralUtilitiesAPI()
```
Comment (0xIslamTaha) [inline ui_testing/testcases/base_test.py]: Rebase from the latest master

File: ui_testing/testcases/base_test.py [hunk 2]
```diff
@@ -3,9 +3,29 @@
...
+from api_testing.apis.article_api import ArticleAPI
+from api_testing.apis.test_plan_api import TestPlanAPI
+from ui_testing.pages.header_page import Header
+from api_testing.apis.orders_api import OrdersAPI
+from ui_testing.pages.analysis_page import SingleAnalysisPage
+from api_testing.apis.contacts_api import ContactsAPI
+from api_testing.apis.users_api import UsersAPI
+from api_testing.apis.roles_api import RolesAPI
+from api_testing.apis.analysis_api import AnalysisAPI
+from api_testing.apis.general_utilities_api import GeneralUtilitiesAPI
```
[the remark this answers is not in the document]
Comment (omarkasem) [reply ui_testing/testcases/base_test.py]: Fixed

**Pull request merged**